2019_01_29_Map_Collections_Two/src/Main.java

```diff
@@ -0,0 +1,38 @@
+import java.util.ArrayList;
+import java.util.Collections;
+import java.util.HashMap;
+import java.util.Map;
+
+public class Main {
+
+	public static void main(String[] args) {
+		
+		ArrayList<String> list = new ArrayList<>();
+		Collections.addAll(list, "a","b","c","a","s","c","d");
+		System.out.println(list);
+		System.out.println(returnMapKeys(list) );
+
+	}
+	
+	public static Map<String, Boolean> returnMapKeys(ArrayList<String> list){
+		
+		Map<String, Boolean> res = new HashMap <String, Boolean>();
+		int count = 0;
+		
+		for (String s:list) {
+			count = Collections.frequency(list, s);
```

Comment:
Can you come up with a solution, which does not use `Collections.frequency()`?
Hint: all elements of the array should be in the resulting map, but with different values per key, right?

Reply:
Though in general it is good to use a library function. But practice and come with a solution using maps only is good.